content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
 
-.. image:: reporting/meal-vouchers.png
-   :align: center
-   :alt: Meal voucher
+To change the displayed view, click the :guilabel:`➕ (plus)` icon next to the word
+:guilabel:`Total`, then click on one of the grouping options. The available options are
+:guilabel:`Work Entry Type`, :guilabel:`Employee`, and :guilabel:`Department`. If in a multi-company
+database, a :guilabel:`Company` option also appears.
 
-To change the displayed view, click on :guilabel:`Total`. The data collapses, showing only the count
-column. Click on :guilabel:`Total` again, then hover over :guilabel:`Day`, then click on one of the
-other time-period options available.
+To add a new group to sort the data by, click :guilabel:`Add Custom Group`, then click one of the
+presented options.
 
-.. image:: reporting/voucher-day.png
-   :align: center
-   :alt: Meal voucher
+.. tip::
+   Wherever a :guilabel:`➕ (plus)` icon appears on a pivot table, the information can be further
+   grouped. Click on a :guilabel:`➕ (plus)` icon to reveal the available grouping options.
 
-It is possible to compare the current meal voucher report to the one for the previous time period or
-the previous year. To view these comparisons, click on the :guilabel:`Comparison` drop-down menu at
-the top, then select either :guilabel:`Date: Previous Period` or :guilabel:`Date: Previous Year`.
+   Click on a :guilabel:`➖ (minus)` icon anywhere on the pivot table to remove that grouping.
 
-.. image:: reporting/meal-comparison.png
-   :align: center
-   :alt: Meal voucher
+It is possible to compare the current :guilabel:`Work entry analysis` report to the previous month
+or the previous year. To view these comparisons, click the :guilabel:`drop-down arrow` in the search
+bar to reveal the various filter and grouping options.
 
-To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information will
-be downloaded into an Excel spreadsheet.
+In the section titled :guilabel:`Comparison`, click on either :guilabel:`Current Month: Previous
+Period` or :guilabel:`Current Month: Previous Year`. The report updates and displays the previous
+time period values, as well as the :guilabel:`Variation` between the two.
 
-.. image:: reporting/voucher-download.png
+.. image:: reporting/work-entry-comparison.png
    :align: center
-   :alt: Download an Excel spreadsheet of the data by clicking the download button.
+   :alt: A pivot table comparing the work entries of the current month and the previous month.
 
-The data can be inserted into a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet`
-button. A pop-up appears asking which spreadsheet to place the information in. Select an existing
-spreadsheet or enter the name for a new spreadsheet. Click the :guilabel:`Confirm` button to move to
-a spreadsheet view with all the information populated.
+To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information is
+downloaded into a spreadsheet.
```

Comment:
again, a little more guidance for the reader here would prove beneficial, as they likely aren't aware of what a 'Download xlsx' icon looks like.